Neos.Flow/Classes/Property/TypeConverter/CollectionConverter.php

```diff
@@ -85,6 +86,9 @@ public function getSourceChildPropertiesToBeConverted($source)
     public function getTypeOfChildProperty($targetType, $propertyName, PropertyMappingConfigurationInterface $configuration)
     {
         $parsedTargetType = TypeHandling::parseType($targetType);
+        if ($parsedTargetType['elementType'] === null) {
+            throw new InvalidDataTypeException(sprintf('The collection type property "%s" is missing an element type (got "%s")', $propertyName, $targetType), 1600530641);
```

Comment:
I'm not entirely sure what this "collecction type property" exception tells me. What is the "collection type property" in this example? Is it the inside of the Collection annotation (`Collection<$elementType>`)?

Can we rephrase to 

"The annotated Collection property "%s" was not given a element type. Should be given in the format Collection<$elementType> and it ususally a class name of a different object"

Reply:
```suggestion
            throw new InvalidDataTypeException(sprintf('The annotated collection property "%s" was not given an element type. Should be given in the format Collection<$elementType> and is ususally a class name of a different object.', $propertyName, $targetType), 1600530641);
```